src/DIRAC/Resources/Computing/ARCComputingElement.py

```diff
@@ -63,14 +63,22 @@ def __init__(self, ceUniqueID):
     self.mandatoryParameters = MANDATORY_PARAMETERS
     self.pilotProxy = ''
     self.queue = ''
-    self.outputURL = 'gsiftp://localhost'
     self.gridEnv = ''
     self.ceHost = self.ceName
+    self.endpointType = 'Gridftp'
     self.usercfg = arc.common.UserConfig()
     # set the timeout to the default 20 seconds in case the UserConfig constructor did not
     self.usercfg.Timeout(20)  # pylint: disable=pointless-statement
     self.ceHost = self.ceParameters.get('Host', self.ceName)
     self.gridEnv = self.ceParameters.get('GridEnv', self.gridEnv)
+
+    # ARC endpoint types (Gridftp, Emies)
+    endpointType = self.ceParameters.get('EndpointType', self.endpointType)
```

Comment:
Would this be yet another CS flag? In that case, it should be documented.

Reply:
Alright, I moved the documentation from `docs` to the `ARCComputingElement` module itself as suggested by @andresailer and I added information about `EndpointType`.